### PR TITLE
Fix model format for optimized model

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
- Set model format to PYTORCH from OPENVINO
- Remove using undefined variable in `config.py`

### CODE REVIEW

**Comment by AlexanderDokuchaev** [comment]
CI-548 ✔️

File: mmdet/apis/ote/apis/detection/nncf_task.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def optimize(
         self.save_model(output_model)
 
         output_model.model_status = ModelStatus.SUCCESS
-        output_model.model_format = ModelFormat.OPENVINO
+        output_model.model_format = ModelFormat.PYTORCH
```
Comment (druzhkov-paul) [inline mmdet/apis/ote/apis/detection/nncf_task.py]: Please update the SDK.

**Review by druzhkov-paul — APPROVED** [review]

File: mmdet/apis/ote/apis/detection/nncf_task.py (hunk 1 above)
Comment (AlexanderDokuchaev) [reply mmdet/apis/ote/apis/detection/nncf_task.py]: done

**Review by druzhkov-paul — APPROVED** [review]
Conditionally approve.

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**